.github/workflows/android.yml

```diff
@@ -0,0 +1,77 @@
+---
+
+on:
+  pull_request:
+    paths:
+      - 'android_beta/*.yml'
+      - 'android_release/*.yml'
+
+jobs:
+  render_notes:
+    name: Render notes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          path: notes
+
+      - name: Checkout render-notes.py
+        uses: actions/checkout@v4
+        with:
+          repository: kewisch/thunderbird-android
```

Comment:
I think this needs to point to the thunderbird repository

Reply:
Good catch!